README.md

```diff
@@ -30,36 +29,36 @@ You will store these objects in memory for the time being. Meaning that if the *
 
 **Routes overview table:**
 
-| **METHOD** **ROUTE (uri)** | **BODY** | **RESPONSE** | **Action** |
+| **METHOD** **ROUTE (URI)** | **BODY** | **RESPONSE** | **Action** |
 |---|---|---|---|
 | `GET /` | empty | render HTML `index.html` | serves the `index.html` |
 | `GET /buzzwords` | empty | `{ "buzzWords": [...] }` A JSON response containing an array of current buzz words | Retrieves all buzzwords |
-| `POST /buzzword` | `{ "buzzWord": String, "points": Number }` | `{ "success": true }` | Creates a new buzzword object. Returns `true` if successful else `false`|
-| `PUT /buzzword` | `{ "buzzWord": String, "heard": Bool }` |  `{ "success": true, newScore: Number }` | Updates a buzzword. Returns `true` and the new score if successful otherwise returns just `false` |
-| `DELETE /buzzword` | `{ "buzzWord": String }` | `{ "success": true }` | Delete a buzzword. Returns `true` if successful else `false` |
-| `POST /reset` | `{ "reset": true }` | `{ "success": true }` | Resets the server. All buzzwords are removed and scores reset to `0` |
+| `POST /buzzwords` | `{ "buzzWord": String, "points": Number }` | `{ "success": true }` | Creates a new buzzword object. Returns `true` if successful else `false`|
+| `PUT /buzzwords` | `{ "buzzWord": String, "heard": Bool }` |  `{ "success": true, newScore: Number }` | Updates a buzzword's heard value. This also marks that a buzzword has been heard and should update the total score. Returns `true` and the new total score if successful otherwise returns just `false` |
```

Comment:
`PUT` to `/buzzword` should modify the existing buzzword model, not score the buzzword. A new endpoint should be created for that.
`newScore:` should be changed to `totalScore`